packages/@expo/cli/src/start/startAsync.ts

```diff
@@ -74,9 +74,11 @@ export async function startAsync(
   const platformBundlers = getPlatformBundlers(exp);
 
   if (!options.forceManifestType) {
-    const easUpdatesUrlRegex = /^https:\/\/(staging-)?u\.expo\.dev/;
-    const isEasUpdatesUrl = exp.updates?.url ? easUpdatesUrlRegex.test(exp.updates.url) : false;
-    options.forceManifestType = isEasUpdatesUrl ? 'expo-updates' : 'classic';
+    const classicUpdatesUrlRegex = /^https:\/\/(staging-)?exp\.host/;
+    const isClassicUpdatesUrl = exp.updates?.url
```

Comment:
Probably should just change the regex to match the legacy classic updates hostname:

1. If no updates.url -> new
2. If updates.url matches classic URL -> classic

This would enable users to explicitly indicate that they want to use the classic. Also do `expo-dev-clients` and Expo Go support this new format?

Reply:
Yes, both support this format, and have for about a year. I can make the regex select the `hostname`